samples/modules/thrift/hello/server/Makefile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Meta
+# SPDX-License-Identifier: Apache-2.0
+
+.PHONY: all clean
+
+CXXFLAGS :=
+CXXFLAGS += -std=c++17
+
+GEN_DIR = gen-cpp
+GENSRC = $(GEN_DIR)/Hello.cpp $(GEN_DIR)/Hello.h $(GEN_DIR)/hello_types.h
+GENHDR = $(filter %.h, $(GENSRC))
+GENOBJ = $(filter-out %.h, $(GENSRC:.cpp=.o))
+
+THRIFT_FLAGS :=
+THRIFT_FLAGS += $(shell pkg-config --cflags thrift)
+THRIFT_FLAGS += -I$(GEN_DIR)
+THRIFT_FLAGS += -I/opt/homebrew/Cellar/boost/1.80.0/include
+THRIFT_FLAGS += -I/opt/homebrew/Cellar/openssl@1.1/1.1.1s/include
```

Comment:
This does not seem to be portable at all.

Reply:
Yes, let me remove those two lines.